gradle.properties

```diff
@@ -16,7 +16,6 @@ POM_INCEPTION_YEAR=2016
 android.useAndroidX=true
 android.enableJetifier=false
 
-org.gradle.daemon=true
```

Comment:
Note: This _defaults_ to true. There is no need to set it (it was only required for an older Gradle version).